src/mount.coffee

```diff
@@ -78,6 +84,45 @@ wrapOutport = (transactions, client, instance, port, queueName) ->
     client.sendTo 'outqueue', queueName, data, (err) ->
       debug 'sent output data', queueName, err, data
 
+# Execute all packets using the existing NoFlo network
+wrapPortsOnExisting = (transactions, client, instance, definition) ->
+  for port in definition.inports
+    continue unless port.queue
+    continue unless port.id
+
+    # Normal mode is to reuse same network for each message
+    onMessage = wrapInport transactions, instance, port.id
+    client.subscribeToQueue port.queue, onMessage, (err) ->
+      throw err if err
+
+  for port in definition.outports
+    wrapOutport transactions, client, instance, port.id, port.queue, ->
+
+# Execute each packet using a dedicated NoFlo network
+wrapPortsDedicated = (transactions, client, loader, definition, options) ->
+  definition.inports.forEach (port) ->
+    return unless port.queue
+    return unless port.id
+
+    onMessage = (msg) ->
+      loadAndStartGraph loader, options.graph, options.iips, (err, instance) ->
+        if err
```

Comment:
This sounds like a pretty bad error... Should we just throw so it will be fatal?